tests/client/core/api/test_api.js

```diff
@@ -7,18 +7,26 @@ import { ErrorHandler } from 'core/errorHandler';
 import { unexpectedSuccess } from 'tests/client/helpers';
 
 
+const defaultHeaders = {
+  'Content-Type': 'application/json',
+};
+
+function generateHeaders(headerData = defaultHeaders) {
+  return {
+    get: (name) => headerData[name],
```

Comment:
I went to try this in Safari since that's the only other browser I have installed right now neither `fetch` nor `Response` is supported. So, maybe for all other browsers it works as expected because it isn't supported? 😜